### PR TITLE
Add support for sssa error handling

### PR DESCRIPTION
**Pull request by drrzmr**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 language: go
 
 go:
-  - 1.6
```
Comment (DennyLoko) [inline .travis.yml]: I would suggest splitting this PR into two:
- One with the new Travis configuration
- Other with the code changes

Comment (drrzmr) [inline .travis.yml]: I agree!
This PR stay with code changes, next one I send new Travis config.

File: Makefile [hunk 2]
```diff
@@ -1,15 +1,15 @@
-deps:
-	@go get golang.org/x/tools/cmd/cover
+trevis-deps:
```
Comment (derekstavis) [inline Makefile]: `travis-deps` no `trevis-deps`

File: ckcli/main.go [hunk 3]
```diff
@@ -6,7 +6,7 @@ import (
 	"io"
 	"os"
 
-	"github.com/docopt/docopt.go"
+	"github.com/docopt/docopt-go"
```
Comment (derekstavis) [inline ckcli/main.go]: ?

File: Makefile (hunk 2 above)
Comment (drrzmr) [inline Makefile]: thanks!

File: ckcli/main.go (hunk 3 above)
Comment (drrzmr) [inline ckcli/main.go]: I guess that atom make this :(, fixed!

Comment (DennyLoko) [inline ckcli/main.go]: Well, from the perspective of the project maintainer, it should stay as `docopt-go` as per https://github.com/docopt/docopt.go#installation.

Comment (derekstavis) [inline ckcli/main.go]: Yeah, you are right :+1:

Comment (drrzmr) [inline ckcli/main.go]: great, I will split this change to another patch

**Pull request closed without merging**